### PR TITLE
Pr

### PR DESCRIPTION
**Pull request by AlexKukhtinDev**
Loaded project correct check style

### CODE REVIEW

File: src/main/java/com/akukhtin/internetshop/controller/GetAllItemsController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import com.akukhtin.internetshop.service.ItemService;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetAllItemsController extends HttpServlet {
```
Comment (Sofasmile) [inline src/main/java/com/akukhtin/internetshop/controller/GetAllItemsController.java]:
```suggestion
public class AddItemController extends HttpServlet {
```

File: src/main/java/com/akukhtin/internetshop/controller/GetBucketController.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetBucketController extends HttpServlet {
```
Comment (Sofasmile) [inline src/main/java/com/akukhtin/internetshop/controller/GetBucketController.java]:
```suggestion
public class GetItemsFromBucketController extends HttpServlet {
```

File: src/main/java/com/akukhtin/internetshop/service/serviceimpl/OrderServiceImpl.java [hunk 3]
```diff
@@ -38,6 +38,17 @@ public void delete(Long id) {
...
     }
 
+    @Override
+    public List<Order> getAll() {
+        return orderDao.getAll();
+    }
+
+    @Override
+    public List<Item> getAllItems(Long orderId) {
+        Order order = orderDao.get(orderId);
```
Comment (Sofasmile) [inline src/main/java/com/akukhtin/internetshop/service/serviceimpl/OrderServiceImpl.java]: return orderDao.get(orderId).getItems();

File: src/main/java/com/akukhtin/internetshop/service/serviceimpl/UserServiceImpl.java [hunk 4]
```diff
@@ -37,4 +39,9 @@ public void delete(Long id) {
     public List getOrders(Long userId) {
         return userDao.get(userId).getOrders();
     }
+
+    @Override
+    public List<User> getAll() {
+        return Storage.users;
```
Comment (Sofasmile) [inline src/main/java/com/akukhtin/internetshop/service/serviceimpl/UserServiceImpl.java]: we can't use Storage in service layer

File: src/main/java/com/akukhtin/internetshop/Main.java [hunk 5]
```diff
@@ -16,6 +20,7 @@
 public class Main {
```
[the remark this answers is not in the document]
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/Main.java]: If i delete this class , not will be worked Injector

File: src/main/java/com/akukhtin/internetshop/controller/GetAllItemsController.java (hunk 1 above)
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/controller/GetAllItemsController.java]: ok

File: src/main/java/com/akukhtin/internetshop/controller/GetBucketController.java (hunk 2 above)
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/controller/GetBucketController.java]: ok

File: src/main/java/com/akukhtin/internetshop/service/serviceimpl/OrderServiceImpl.java (hunk 3 above)
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/service/serviceimpl/OrderServiceImpl.java]: Okey

File: src/main/java/com/akukhtin/internetshop/service/serviceimpl/UserServiceImpl.java (hunk 4 above)
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/service/serviceimpl/UserServiceImpl.java]: OK

File: src/main/java/com/akukhtin/internetshop/Main.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/Main.java]: okey delete

**Pull request merged**